finder-app/writer.c

```diff
@@ -0,0 +1,59 @@
+
+/*
+ *  
+ *  Author     : Vignesh Vadivel
+ *  email      : viva9969@colorado.edu 
+ *  Course     : Advanced Embedded Software Development
+ *  Assignment : 02
+ * 
+ */
+
+/***************************************** HEADER_FILES *******************************************************/
+#include <syslog.h>
+#include <fcntl.h>
+#include <stddef.h>
+#include <unistd.h>
+#include <string.h>
+
+/***************************************** Main Function *******************************************************/
+int main(int argc, char* argv[]){
+  if (argc!=3){                                                                  // Check valid number of arguments //
+    openlog(NULL, 0, LOG_USER);                                                  // Open Log file //
+    syslog(LOG_ERR, "Invalid Number of Arguments: %d\n", argc);                  // Log the error message //
```

Comment:
@Vignesh-Vadivel adding argument detail here would be helpful.

Reply:
Sure Dan. I will improve